app/assets/javascripts/modules/image_x_viewer.js

```diff
@@ -253,7 +253,10 @@
     var _setupButtonListeners = function() {
       $embedHeader.find('[data-sul-view-fullscreen="fullscreen"]')
         .on('click', function() {
+          var worldRestriction = $('[data-world-restriction]').data('world-restriction');
```

Comment:
:+1: Good call!

I think that was our first attempt at trying to figure out how to disable in full-screen, and when we figured out that we needed to disable the context menus scoped to a different element in full-screen, didn't go back to see if this was ever necessary.